src/dynode.h

```diff
@@ -203,8 +203,8 @@ class CDynode : public dynode_info_t
 
     bool UpdateFromNewBroadcast(CDynodeBroadcast& dnb, CConnman& connman);
 
-    static CollateralStatus CheckCollateral(const COutPoint& outpoint);
-    static CollateralStatus CheckCollateral(const COutPoint& outpoint, int& nHeightRet);
+    static CollateralStatus CheckCollateral(const COutPoint& outpoint, const CPubKey& pubkey);
```

Comment:
What version or branch of Dash did this come from?

Reply:
develop branch of Dash

Reply:
https://github.com/dashpay/dash/pull/1783/files